### ISSUE TITLE
Cannot install package in atom 1.18.0 x64, Ubuntu 16.04

### ISSUE THREAD

**Issue by PaulSmith220** [abe33/atom-pigments#317]
Clicking "install", installation failed with following message:

`Installing “pigments@0.39.1” failed.Hide output…

npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! Linux 4.8.0-56-generic
npm ERR! argv "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/home/paul/.atom/.apm/.apmrc" "--userconfig" "/home/paul/.atom/.apmrc" "install" "/tmp/d-117523-3224-6fn9dv.qut2t65hfr/package.tgz" "--runtime=electron" "--target=1.3.15" "--arch=x64" "--global-style"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! file /home/paul/.atom/.apm/fs-plus/2.10.1/package/package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! No data, empty input at 1:1
npm ERR! 
npm ERR! ^
npm ERR! File: /home/paul/.atom/.apm/fs-plus/2.10.1/package/package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR! 
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/apm-install-dir-117523-3224-1odwufa.3cz38cwhfr/npm-debug.log`

**Comment by C-Bouthoorn** [comment]
It seems the download was corrupted. Could you try removing the plugin (also removing it from the `.apm` folder) and re-installing?

**Comment by PaulSmith220** [comment]
Yes, i've already did it (removed it from "packages" and from ".apm"), same result.
I fixed that error with fs-plus package.json by adding this file manually from GitHub.

But this error occurs:
`npm ERR! cb() never called!`
And no more text, only this row.
I also tried to install it manually with _apm install pigments_ and _apm install_ in package directory, same error.

**Comment by C-Bouthoorn** [comment]
Does this still persist on the latest version on Pigments?
